mis_builder/models/prorata_read_group_mixin.py

```diff
@@ -60,30 +60,30 @@ def read_group(
         if (
             date_from is not None
             and date_to is not None
-            and not any(":" in f for f in flds)
+            and not any(":" in f for f in fields)
         ):
-            dt_from = fields.Date.from_string(date_from)
-            dt_to = fields.Date.from_string(date_to)
+            dt_from = ffields.Date.from_string(date_from)
+            dt_to = ffields.Date.from_string(date_to)
             res = {}
-            sum_fields = set(flds) - set(groupby)
-            read_fields = set(flds + ["date_from", "date_to"])
+            sum_fields = set(fields) - set(groupby)
+            read_fields = set(fields + ["date_from", "date_to"])
             for item in self.search(domain).read(read_fields):
                 key = tuple(item[k] for k in groupby)
                 if key not in res:
                     res[key] = {k: item[k] for k in groupby}
                     res[key].update({k: 0.0 for k in sum_fields})
                 res_item = res[key]
                 for sum_field in sum_fields:
-                    item_dt_from = fields.Date.from_string(item["date_from"])
-                    item_dt_to = fields.Date.from_string(item["date_to"])
+                    item_dt_from = ffields.Date.from_string(item["date_from"])
```

Comment:
Maybe `from odoo.fields import Date` to avoid this `ffields`?

Reply:
sure, it sounds better with an import